package.json

```diff
@@ -80,6 +80,7 @@
     "babel-eslint": "7.2.3",
     "babel-istanbul": "^0.12.2",
     "babel-loader": "7.1.2",
+    "babel-polyfill": "6.26.0",
```

Comment:
Why did you need this? Only for the tests?

Reply:
This was required for the tests indeed, the parsing failed due to missing Promise support.